server/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -700,8 +720,14 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
         final boolean verbose = params.paramAsBoolean("verbose", GetSnapshotsRequest.DEFAULT_VERBOSE_MODE);
         // write snapshot info for the API and any other situations
         builder.startObject();
+        final SnapshotId snapshotId = snapshot.getSnapshotId();
         builder.field(SNAPSHOT, snapshotId.getName());
         builder.field(UUID, snapshotId.getUUID());
+
+        if (params.paramAsBoolean("include_repository", true)) {
+            builder.field(REPOSITORY, snapshot.getRepository());
```

Comment:
I don't see YAML REST tests for this new field - maybe add at least one?

Reply:
++ let's see what we come down on in terms of how this parameter is going to behave. If we just enable it by default even for single repo requests then we can  (and have to in case of docs tests) just add it to existing tests. If we don't want it on by default, then I'll add a new test with multiple repo requests that check for this field :)